modules/openapi-generator/src/test/java/org/openapitools/codegen/options/DartDioClientOptionsProvider.java

```diff
@@ -37,6 +37,7 @@ public class DartDioClientOptionsProvider implements OptionsProvider {
     public static final String PUB_AUTHOR_VALUE = "Author";
     public static final String PUB_AUTHOR_EMAIL_VALUE = "author@homepage";
     public static final String PUB_HOMEPAGE_VALUE = "Homepage";
+    public static final String API_CLIENT_NAME_VALUE = "Homepage";
```

Comment:
This change allows `apiClient` to be used in `additionalProperties` which will override the default name for the top level `ApiClient` class.